app-containers/skopeo/files/makefile-1.13.3.patch

```diff
@@ -0,0 +1,31 @@
+--- a/Makefile
```

Comment:
This patch needs a description at the top saying:
- what it does
- why it does it

Reply:
others install: removals are there because it tries to build already built system in install phase. 

make install should just install not compile

Reply:
Explain that at the top of the patch?

Reply:
done